### PR TITLE
Remove bower and upgrade dependencies

### PR DESCRIPTION
**Pull request by nickjamesio**
This PR depends on #66 

Removed bower from project and upgraded a number of packages to a more recent version. I also removed the reading-time dependency since ghost v2 has support to display estimated reading time.

Lastly, I did not upgrade the bourbon package. I'm not quite sure how to use it or what it really does so I left it alone. I figure it can be upgraded in a future PR. You can let me know your thoughts.

### CODE REVIEW

**Comment by kelyvin** [comment]
The more I look at this gulpfile, the more I realize how far we've come and how much more seamless webpack is with bundling client-side assets. This project was originally forked and started WAY before webpack was the defacto standard and way before using npm for everything was ideal. I see that your changes have some workarounds to make it all work, and I'd like to thank you for making all these changes so far. 

I will approve and merge this pull request. However, I'd actually like to move away from `gulp` and make a switchover to `webpack` for clientside bundling. If that's something you have experience with (and don't mind making the switchover) or something you'd like to learn, please let me know! Otherwise, I'd say the changes you made so far with the current gulp system is great, but I'd shift focus to rearchitecting the client side code instead. I will take it as an action item for myself to switch over to the webpack system to make it easier for developing. 

Thanks!

**Comment by nickjamesio** [comment]
Thanks for the feedback. Working on this project and making some modifications to uno-zen is the first time I've ever used `gulp`. I thought it was a neat tool but I can spend some time looking at `webpack`.

I know people in the react world use `webpack`, but I did not know it had become the defacto bundler. It will take some time for me to get up to speed, but I am up for the challenge. If you know of some good resources or tutorials, please share them :) Or if you want to give me a quick run down on why `webpack` is better, I'm all ears.

Lastly, should I hold off on adding ES6 syntax until we have a basic `webpack` configuration to add `babel` support? I was planning on just using `@babel/preset-env` by itself.

**Comment by kelyvin** [comment]
@nickjamesio 

I'm not exactly sure when webpack became the defacto bundler, but if I were to guess, it was because it just worked out of the box with little configuration. For most use cases, that "out of the box" configuration was good enough - it gets you set up with Babel, live reloading, hot module swapping, minification, bundling, etc. It's pretty sweet

The main negative to webpack is that it's a pretty steep learning curve if you ever need to do anything outside of its default setup. This is mostly because it's configuration-driven development, i.e. it requires sifting through lots of docs. Because it's gone through 4 versions now, you may find yourself looking at outdated advice/tutorials if you run into any issues. Also because it's well optimized for client-side code, if you ever want to use any of its features for Node.js server-side development, it's bit overkill. In fact, my full-stack node.js projects use both - gulp for server-side development and complex tasks, webpack for pure client-side development.

However, I'd say its worth the transition. This theme project is very simple so the default webpack setup should actually do most of what we need. Plus, learning webpack is probably more useful to add to your skillsets long term because (in my opinion) it's the most popular and the longest surviving JS bundler so far (i.e. it started to get mass wide-scale enterprise use around 2015 and is still the most popular). Granted, 3 years isn't that long, but the front-end world changes PRETTY fast, so anything that lasts more than 3 years is worth learning. 

**TL;DR** 
1. If you switch to the basic webpack configuration, babel support should be easy to setup and come out of the box, so you can start ES6 work right away
2. As for tutorials, I found this one to be very thorough in explaining all the core parts, without confusing the heck out of you: https://hackernoon.com/a-tale-of-webpack-4-and-how-to-finally-configure-it-in-the-right-way-4e94c8e7e5c1 

I've also gained extensive experience with webpack over the last two years, so if you have any questions, let me know!

**Comment by kelyvin** [comment]
I'm a night owl. It's a real problem....
